### ISSUE TITLE
bug??

### ISSUE THREAD

**Issue by hhgyu** [SubtitleEdit/subtitleedit#1021]
hhgyu/subtitleedit@e1e1169965fb751ad5cc03c5b72ea8101d9129c2

http://www.filedropper.com/1_20

smi open (program freezing)

 ------------------------

hhgyu/subtitleedit@c487a8b906abc327ee7a47ca18fe377515c641e3

http://www.filedropper.com/showdownload.php/2_52

```
1화 <font color=#FF8200>[늘어져라, 낫토 오크라 덮밥.<BR>
치즈 토핑 도시락. 440kcal]
```
smi to ass -> style(color) do not apply to the conversion.

error
```
1화 [늘어져라, 낫토 오크라 덮밥.\N치즈 토핑 도시락. 440kcal]
```

success
```
1화 {\c&H0082FF&}[늘어져라, 낫토 오크라 덮밥.\N치즈 토핑 도시락. 440kcal]
```

 ------------------------

hhgyu/subtitleedit@d3d629d57a6ec5a371659d597b637b575e2ae8be

jwplayer font style even when the initialization is not properly play the next subtitle

**Comment by niksedk** [comment]
I don't think the filedropper links are working... I could not find any smi files - could you supply new links?

**Comment by hhgyu** [comment]
http://www.filedropper.com/smi

2.smi -> http://www.filedropper.com/1_20

1.smi -> http://www.filedropper.com/showdownload.php/2_52

**Comment by hhgyu** [comment]
https://drive.google.com/file/d/0B7BB5bK9QT0Wdk50TFB0WThMZms/view?usp=sharing

http://www.filedropper.com/1_20 -> 2.smi

http://www.filedropper.com/showdownload.php/2_52 -> 1.smi

**Comment by niksedk** [comment]
Thx, got the files from google drive...

The freezing should be fixed in this commit: 0510a25d038575b51cce37efc9864df7d2efb451
(could actually happen with any large data inside smi files)
